docs/docs/FAQ/Machine-Learning-FAQ.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 6
+---
+# Machine Learning
+
+### How does smart search work?
+
+Immich uses CLIP models, for more information about CLIP and its capabilities can be [read here](https://openai.com/research/clip).
```

Comment:
"[...] capabilities read about it here"

Reply:
Shouldn't it be with a dot between them?

"[...] capabilities. read about it here."

Reply:
Imo no, because it still references the "for". Maybe we can rephrase it though: "For more information [...], continue reading here"

Reply:
yes my mistake.
fixed.
Thanks!